proposals/ml_fit_class_xgboost.json

```
@@ -0,0 +1,155 @@
+{
+    "id": "ml_fit_class_xgboost",
+    "summary": "Train an XGBoost classification model",
+    "description": "Executes the fit of an XGBoost classification model based on training data.",
```

"Execute the fit of" sounds a bit convoluted to me, why not just
```suggestion
    "description": "Fit an XGBoost classification model to training data.",
```

I would also add a 1or 2-sentence description of XGBoost itself.

---

Noted.